ultra/tests/test_curriculum.py

```diff
@@ -162,10 +161,11 @@ def test_static_train_single_agent(self):
     def test_static_coordinator(self):
         log_dir = os.path.join(CurriculumTest.OUTPUT_DIRECTORY, "logs/")
         curriculum_dir = "../tests/scenarios/static_test_curriculum"
-        num_episodes = 8
+        num_episodes = 6
         etag = "sac-v0"
 
         agent_coordinator = Coordinator(curriculum_dir, num_episodes)
+        agent_coordinator._train_grade = ">>> TEST COORDINATOR <<<"
```

Comment:
Is this a placeholder? it seems like later on in the code you have a dict or list `self._train_grade = CurriculumInfo.curriculum["static"]["grades"][counter]`

Reply:
Yes, the problem was that the `display()` in coordinator.py uses the `self._grade_counter`. However, in the test cases, I forgot to admit the agent into curriculum (initial `next_train_grade`). Because of that the `self._grade_counter` was not intialized and it produced an error. I have updated the code and the new changes removes line 168